src/assets/locales/locale_config.js

```diff
@@ -15,5 +15,6 @@ export const FALLBACK_LOCALES = {
   "zh-hans": "zh",
   "zh-hans-cn": "zh",
   pt: "pt-br",
+  "pt-pt": "pt-br",
```

Comment:
He have updated the code to handle the sub-language tags so we don't need this anymore. Rebase from master and keep the current master code when rebasing. Let me know if you have issues with that and I can rebase and create a new PR for this.